components/brave_shields/browser/tracking_protection_service.cc

```diff
@@ -190,25 +186,15 @@ void TrackingProtectionService::UpdateFirstPartyStorageTrackers(
       base::flat_set<std::string>(std::move(storage_trackers));
 }
 
-#endif
-
-bool TrackingProtectionService::ShouldStoreState(BraveCookieSettings* settings,
-                                                 HostContentSettingsMap* map,
+#else  // !BUILDFLAG(BRAVE_STP_ENABLED)
+bool TrackingProtectionService::ShouldStoreState(HostContentSettingsMap* map,
```

Comment:
we can save a couple of lines by keeping a single function header and putting `#ifdef`s inside

Reply:
well, I think the buildflags need to be changed in general for `BRAVE_STP_ENABLED` because it's hard to follow what's going on. Probably move all real STP code into a separate `.cc` file and keep this method here

Reply:
@jumde wdyt? maybe file an issue?